test/beacon/Voucher.test.ts

```diff
@@ -352,6 +352,49 @@ describe('Voucher', function () {
             expect(await voucherAsOwner.getSponsoredAmount(dealId)).to.be.equal(dealPrice);
         });
 
+        it('Should match orders with partial sponsored amount', async () => {
+            await addEligibleAssets([app, dataset]); // not workerpool.
```

Comment:
Should we merge classic and boost tests when possible?

Reply:
Boost test was already implemented and there is a TODO to refactor boost and classic tests.

Reply:
Yes :)